keras/backend/mxnet_backend.py

```diff
@@ -1332,9 +1346,19 @@ def normalize_batch_in_training(x, gamma, beta,
     # Returns
         A tuple length of 3, `(normalized_tensor, mean, variance)`.
     """
-    raise NotImplementedError
+    if isinstance(x, KerasSymbol):
+        x = x.symbol
+    normal, mean, var = mx.sym.BatchNorm(data=x, gamma=gamma.symbol, beta=beta.symbol, output_mean_var=True)
+    return KerasSymbol(normal), KerasSymbol(mean), KerasSymbol(var)
 
 
+def batch_normalization(x, mean, var, beta, gamma, epsilon=1e-3):
+    """Apply batch normalization on x given mean, var, beta and gamma.
+    """
+    if isinstance(x, KerasSymbol):
+        x = x.symbol
+    return KerasSymbol(mx.sym.BatchNorm(x))
```

Comment:
should be mx.sym.BatchNorm(data=x, gamma=gamma, beta=beta, moving_mean=mean, moving_var=var)